src/components-examples/angular/pagination/index.ts

```diff
@@ -0,0 +1,29 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { SbbButtonModule } from '@sbb-esta/angular-public/button';
```

Comment:
Replace the `@sbb-esta/angular-public` modules with the ones from `@sbb-esta/angular`

Reply:
When changing the module imports, you also need to slightly adapt the usage: e.g. sbbButton to sbb-button
Also run `yarn baz` again to ensure the correct dependencies are listed for bazel.